### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by chaytanyasinha**
What kind of change does this PR introduce?
fix

currently, the README.md doesn't have important thing that should have been mentioned for webpack as a JavaScript module bundler. 

Did you add tests for your changes?
NA

Does this PR introduce a breaking change?
NO,but will help the new user to learn more about webpack.

What needs to be documented once your changes are merged?
Nothing.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -294,10 +294,10 @@ or are automatically applied via regex from your webpack configuration.
 [riot-npm]: https://img.shields.io/npm/v/riot-tag-loader.svg
 [riot-size]: https://packagephobia.now.sh/badge?p=riot-tag-loader
 
-### Performance
+### [Performance](https://webpack.js.org/configuration/performance/)
 
 webpack uses async I/O and has multiple caching levels. This makes webpack fast
-and incredibly **fast** on incremental compilations.
+and incredibly **fast** on incremental compilations. This feature was inspired by the idea of [webpack Performance Budgets](https://github.com/webpack/webpack/issues/3216).
```
Comment (sokra) [inline README.md:300]: These are two totally different things. runtime performance vs build performance

File: README.md [hunk 2]
```diff
@@ -318,6 +318,16 @@ you full control of what is loaded initially and what is loaded at runtime
 through code splitting. It can also make your code chunks **cache
 friendly** by using hashes.
 
+### [Node](https://webpack.js.org/configuration/node/)
+
+webpack's Node.js options configure whether to polyfill or mock certain Node.js globals and modules. This allows code originally written for the Node.js environment to run in other environments like the browser.
```
Comment (sokra) [inline README.md:323]: This has been removed in this version of webpack.

**Comment by chaytanyasinha** [comment]
@sokra I have updated the pull request.Please review.

**Comment by himanshu010** [comment]
CI failed, fix it first.

**Comment by chaytanyasinha** [comment]
@himanshu010  Can you please help me with that,I have tried to do but it's not working.

**Comment by himanshu010** [comment]
I think you forget to run prettier
run the command: **yarn pretty-lint-fix**

If prettier is not installed, then install it first using:  **yarn global add prettier**
And then run that command

**Comment by chaytanyasinha** [comment]
@rishabh3112 Please review

**Comment by snitin315** [comment]
please reopen your PR for review

**Comment by chaytanyasinha** [comment]
@snitin315 actually because of CI failure even after running all the commands I closed this and thought of adding another request

**Comment by snitin315** [comment]
your branch is too much outdated from upstream (297 commits behind).
First, you need to reset your branch with upstream.
```
git remote add upstream git@github.com:webpack/webpack.git
git fetch upstream master
git reset --hard upstream/master
```
then make your changes again, commit and force push the changes.

`git push origin master --force`

**Comment by snitin315** [comment]
It looks like you pushed before making new changes and commiting them.

**Comment by chaytanyasinha** [comment]
@snitin315 yeah by mistake

**Comment by snitin315** [comment]
now you can push your changes again and open PR.

**Comment by chaytanyasinha** [comment]
@snitin315 I am trying but 
'' 32 problems (32 errors, 0 warnings)
husky > pre-commit hook failed (add --no-verify to bypass)''
this error is comming

**Comment by snitin315** [comment]
You need to fix those errors first.

**Comment by chaytanyasinha** [comment]
Is it because of prettier?

**Comment by snitin315** [comment]
Pre commit hook automatically run prettier/lint for you before each commit. When some errors are not fixable automatically then you have to fix them manually.

**Comment by chaytanyasinha** [comment]
Okay @snitin315 ,were these problems faced by you too while setting up?

**Comment by snitin315** [comment]
No

**Pull request closed without merging**